src/ProjectTemplates/Microsoft.Extensions.AI.Templates/Microsoft.Extensions.AI.Templates.csproj

```diff
@@ -61,6 +61,14 @@
         **\NuGet.config;
         **\Directory.Build.targets;
         **\Directory.Build.props;" />
+    <Content
+      Include="src\McpServer\**\*"
+      Exclude="
+        **\bin\**;
+        **\obj\**;
+        **\.vs\**;
+        **\*.sln;
+        **\*.in;" />
```

Comment:
Could we add a comment about keeping this list in sync with `McpServerSnapshotTests.cs` (similar to the other comment in this file)?